pkgs/misc/cups/drivers/cnijfilter2/default.nix

```diff
@@ -4,11 +4,11 @@
 stdenv.mkDerivation {
   pname = "cnijfilter2";
 
-  version = "5.70";
+  version = "5.90";
 
   src = fetchzip {
-    url = "http://gdlp01.c-wss.com/gds/0/0100009930/01/cnijfilter2-source-5.70-1.tar.gz";
-    sha256 = "045zjsmaidn1m44ki6m1018gjzbj77gm234n5i2lshxpbzpyh0is";
+    url = "http://gdlp01.c-wss.com/gds/4/0100010484/01/cnijfilter2-source-5.90-1.tar.gz";
```

Comment:
Hi Pavol,

happy to use https protocol instead of http, thank you for the notice!

Unfortunately version is not the only string that differs in the url. First url has "0/0100009930/01" string in it and the second has "4/0100010484/01". What is the benefit of having the version in url that must change as a whole when new version is published?

Reply:
Not much. Guess I think it's fine to leave it that way.